tests/env.py

```diff
@@ -12,3 +14,19 @@
 PY2 = sys.version_info.major == 2
 
 PY = sys.version_info
+
+
+def deprecated_call():
+    """
+    pytest.deprecated_call() seems broken in pytest<3.9.x; concretely, it
+    doesn't work on CPython 3.8.0 with pytest==3.3.2 on Ubuntu 18.04 (#2922).
+
+    This is a narrowed reimplementation of the following PR :(
+    https://github.com/pytest-dev/pytest/pull/4104
+    """
+    pieces = pytest.__version__.split(".")
```

Comment:
Working: Will add explicit comment to remove this when min. `pytest` version is required.

Reply:
Done 7cd83686